Contributing-code-to-Oppia.md

```diff
@@ -76,7 +76,8 @@ We also **strongly recommend** looking through the resources under "Developing O
 
 Welcome! Please make sure to follow the instructions above if you haven't already.
 
-After that, you can choose a good first issue from the [list of good first issues](https://github.com/oppia/oppia/labels/good%20first%20issue). These issues are hand-picked to ensure that you don't run into unexpected roadblocks while working on them, and each of them should have clear instructions for new contributors. If you see one that doesn't, please let us know via [GitHub Discussions](https://github.com/oppia/oppia/discussions) and we'll fix it. For other issues, you might need to be more independent because we might not know how to solve them either.
```

Comment:
I think you should keep the last sentence?

Reply:
@seanlip I didn't delete the last sentence. I shifted it to the last line after adding get help line just to make it look more continuous like "good first issues -> github discussions -> get help -> and then finally adding be independent to solve"......this kind of flow.
should I change it?

Reply:
I don't think it makes sense there. That sentence is talking about issues and so is this one; separating it with a sentence talking about running into problems breaks the context (since it's now not clear what "other issues" refers to). So I'd put it back here, and you can start a new paragraph for the "run into problems" bit.

Reply:
Right! I changed it